addons/sys_gui/fnc_inventoryMonitorPFH.sqf

```diff
@@ -0,0 +1,54 @@
+/*
+ * Author: ACRE2Team
+ * Event handler to monitor inventory changes upon ACRE radios.
+ *
+ * Arguments:
+ * 0: Eventhandler [eventhandler, function handler] <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * ADDPFH(DFUNC(inventoryMonitorPFH), 0, []);
```

Comment:
Should probably write it out or something. Like 
`* [acre_sys_gui_inventoryMonitorPFH, 0, []] call CBA_fnc_addPerFrameHandler`
and without final `;`.
Preferably we don't want macros in examples.